### PR TITLE
Add missing cards and update rates in OCA and MCA

### PR DESCRIPTION
**Pull request by latiosu**
* **Addressed Issue(s)**: #5589 

* **Server Mode**: Both

* **Description of Pull Request**: 
Add missing cards to Old Card Album and Mystical Card Album as well as updating rates to match divine-pride.net

### CODE REVIEW

File: db/re/item_cardalbum.txt [hunk 1]
```diff
@@ -374,73 +379,116 @@ IG_CardAlbum,4448,1		// Cornus Card
...
-IG_MagicCardAlbum,4384,1	// Hydro Card
-IG_MagicCardAlbum,4395,3	// Thanatos Maero Card
-IG_MagicCardAlbum,4396,3	// Thanatos Odium Card
-IG_MagicCardAlbum,4397,3	// Thanatos Despero Card
+IG_MagicCardAlbum,4047,3	// Ghostring Card
+IG_MagicCardAlbum,4054,3	// Angeling Card
+IG_MagicCardAlbum,4163,14	// Gryphon Card
+IG_MagicCardAlbum,4169,14	// Dark Illusion Card
+IG_MagicCardAlbum,4174,3	// Deviling Card
+IG_MagicCardAlbum,4179,140	// Dragon Fly Card
```
Comment (Atemo) [inline db/re/item_cardalbum.txt:463]: Shouldn't the rates be 10?

Comment (Latiosu) [reply db/re/item_cardalbum.txt:463]: Ah one thing I forgot to mention was that the rates for `IG_MagicCardAlbum` were based on the Natural Random values listed on DP: 
image

@Atemo Should I be referring to the regular random values instead?
image

**Review by RagnaWay — APPROVED** [review]

File: db/re/item_cardalbum.txt (hunk 1 above)
Comment (Atemo) [reply db/re/item_cardalbum.txt:463]: Yes, the value on rathena should match the normal values

**Comment by Atemo** [comment]
Done in https://github.com/rathena/rathena/pull/6120 Thanks !

**Pull request closed without merging**